### PR TITLE
Jfr edits

### PR DESCRIPTION
**Pull request by JFRudzinski**
Mostly just edits for clarity. Take or leave what you like.

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -49,17 +49,20 @@ graph LR;
...
+Here, "Input structure" refers to the all _input_ material given to perform the calculation (e.g., atomic structure, model, parameters, etc.). "DFT", "TB" and "DMFT" refer to individual _tasks_ of the workflow, which each correspond to a _SinglePoint_ entry in NOMAD. "Output calculation" refers to the _output_ data of each of the final DMFT tasks.
 
-Each task is then parsed into what we call a _SinglePoint_ entry in NOMAD. Our goal is then to set up the following workflows:
+The goal of this tutorial is to set up the following workflows:
 
-1. An example for a `SinglePoint` workflow using the `pressure1` subfolder data. This file can be ignored for later steps in the Tutorial.
-1. A workflow entry for each pressure P<sub>i=1,2</sub> grouping all `SinglePoint` "DFT", "TB", "DMFT at T<sub>1</sub>", and "DMFT at T<sub>2</sub>" tasks.
-2. An extra workflow entry grouping all pressure and all temperature calculations.
+1. A `SinglePoint` workflow for the `pressure1` subfolder data.
```
Comment (JosePizarro3) [inline docs/tutorial.md:56]: Replace by:

 A `SinglePoint` workflow for one of the calculations (e.g., the DFT one) in the `pressure1` subfolder.

**Review by JosePizarro3 — APPROVED** [review]
Thanks a lot for the hard work on styling the tutorial.

I had a couple of questions. Do you want me to take over the branch and make the changes, or do you want to take over?

**Comment by JFRudzinski** [comment]
> Thanks a lot for the hard work on styling the tutorial.
> 
> I had a couple of questions. Do you want me to take over the branch and make the changes, or do you want to take over?

I would say that you can take over the branch since you should have the last say. I will answer your questions and then we can just discuss if anything else comes up?

File: docs/index.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
 # Workflows in NOMAD
 
-Workflows are an important aspect of data management as they enable for a systematic organization of the _tasks_ performed during any Materials Science research project. We refer to _workflow_ to a series of experiments or simulations already performed and composed of _inputs_, _outputs_, and _tasks_ done in a serial or parallel way. Each workflow will have their _entry_ in NOMAD, from which the rest of the meta-information can be accessed via _references_ to other entries and _sections_.
+Workflows are an important aspect of data management as they enable a systematic organization of the _tasks_ performed during any Materials Science research project. We refer to a _workflow_ as a series of experiments or simulations composed of _inputs_, _outputs_, and _tasks_ performed either in serial or in parallel. Each _entry_ in NOMAD may have _workflow_ sections, describing how the (meta)data within the entry was generated. Additionally, an "overarching" workflow can be generated within its own _entry_, to define connections between multiple entries (and subworkflows) via _references_ to the corresponding entries and _sections_.
```
[the remark this answers is not in the document]
Comment (JFRudzinski) [reply docs/index.md:3]: I also wasn't sure. Probably you can just say each entry has a workflow section.

File: docs/tutorial.md [hunk 3]
```diff
@@ -49,17 +49,20 @@ graph LR;
     D11[DMFT at T<sub>1</sub>] --> E11([Output calculation P<sub>1</sub>, T<sub>1</sub>])
     D12[DMFT at T<sub>2</sub>] --> E12([Output calculation P<sub>1</sub>, T<sub>2</sub>])
 ```
-Here "Input structure" refers to the _input_ material on which the calculation is performed, "DFT", "TB" and "DMFT" each of the _tasks_ of our calculation, and "Output calculation" the _output_ calculation of each of the final DMFT tasks.
+Here, "Input structure" refers to the all _input_ material given to perform the calculation (e.g., atomic structure, model, parameters, etc.). "DFT", "TB" and "DMFT" refer to individual _tasks_ of the workflow, which each correspond to a _SinglePoint_ entry in NOMAD. "Output calculation" refers to the _output_ data of each of the final DMFT tasks.
```
[the remark this answers is not in the document]
Comment (JFRudzinski) [reply docs/tutorial.md:52]: Yeah, I think that would be clearer.

File: docs/tutorial.md [hunk 4]
```diff
@@ -94,18 +97,19 @@ workflow2:
...
 
 1. **`name`** keys are optional.
-2. **`inputs`** reference the section containing inputs of the whole workflow. In this case this is the section `run[0].system[-1]` parsed from the mainfile in the path `pressure1/dft_p1.xml`.
-3. **`outputs`** reference the section containing outputs of the whole workflow. In this case this is the section `run[0].calculation[-1]` parsed from the mainfile in the path `pressure1/dft_p1.xml`.
-4. **`tasks`** reference the section containing tasks of each step in the workflow. These must also contain `inputs` and `outputs` properly referencing the corresponding sections; this will then _link_ inputs/outputs/tasks in the NOMAD Archive. In this case this is a `TaskReference` to the section `workflow2` parsed from the mainfile in the path `pressure1/dft_p1.xml`.
-5. **`section`** reference to the uploaded mainfile specific section. The left side of the `#` symbol contains the path to the _mainfile_, while the right contains the path to the _section_.
+2. The root path of the upload can be referenced with `../upload/archive/mainfile/`. Starting from there, the original file structure of the upload is maintained.
```
[the remark this answers is not in the document]
Comment (JFRudzinski) [reply docs/tutorial.md:100]: I don't have a huge preference, I guess I like directory tree structure slightly the best

File: docs/tutorial.md (hunk 1 above)
Comment (JFRudzinski) [reply docs/tutorial.md:56]: 👍

File: docs/tutorial.md [hunk 5]
```diff
@@ -153,15 +157,19 @@ workflow2:
...
   outputs:
     - name: Output DMFT at P1, T1 calculation
       section: '../upload/archive/mainfile/pressure1/temperature1/dmft_p1_t1.hdf5#/run/0/calculation/-1'
     - name: Output DMFT at P1, T2 calculation
       section: '../upload/archive/mainfile/pressure1/temperature2/dmft_p1_t2.hdf5#/run/0/calculation/-1'
 ```
-Now, `tasks` are defined for each of the methodologies performed. If properly referenced, `inputs` and `outputs` inside each `task` will be able to _link_ among them. Then:
+Now, `tasks` are defined for each of the methodologies performed (each corresponding to an underlying SinglePoint workflow). To define a valid workflow, each task must contain an input that corresponds to one of the outputs of the previous task. Moreover, the first task should take as input the overall input of the workflow, and the final task should also have as an output the overall workflow output.
+<!---
+If properly referenced, `inputs` and `outputs` inside each `task` will be able to _link_ among them.
```
[the remark this answers is not in the document]
Comment (JFRudzinski) [reply docs/tutorial.md:170]: 👍

**Pull request merged**